modules/pool/invest/lib/usePoolGetMaxProportionalInvestmentAmount.ts

```diff
@@ -60,10 +50,11 @@ export function usePoolGetMaxProportionalInvestmentAmount() {
             },
         ],
         async ({ queryKey }) => {
-            const hasEth = !!tokenOptionsWithHighestValue.find((token) => isEth(token.address));
             const fixedAmount = {
                 ...tokenWithSmallestValue,
-                address: replaceEthWithWeth(tokenWithSmallestValue.address),
+                address: isEth(tokenWithSmallestValue.address)
```

Comment:
why change this? `replaceEthWithWeth` is a pass through function, its checking internally